debezium-testing/tmt/tests/debezium/test.sh

```diff
@@ -35,6 +35,10 @@ then
     export ORACLE_CONNECTION="-Ddatabase.dbname=FREEPDB1 -Ddatabase.pdb.name=FREEPDB1"
     debezium-testing/tmt/tests/debezium/init-db.sh
   fi
+  DATABASE_USER="c##dbzuser"
+  if [[ "$ORACLE_VERSION" = *noncdb ]]; then
```

Comment:
You may also have to pass `database.pdb.name` with an empty value?

Reply:
this is already passed in another parameter, will try. Thanks!

Reply:
@Naros we are not running noncdb on the TF anyway..